build.sh

```diff
@@ -416,6 +417,58 @@ function build_java()
 
 }
 
+################################################################################
+function clean_documentation()
+{
+    # Remove the content of ${doc_folder}/_build
+    rm -rf ${doc_folder}/_build
```

Comment:
also, clean ./doc

Reply:
It would be the best approach. 
Although If I can only generate html and not pdf. The pdf will failed but the HTML will be generated. 
So, I would not remove /doc here, it will be done before been generate:
```
rm -rf ${generate_doc_folder}/html
mkdir -p ${generate_doc_folder}/html
cp -rf ${doc_folder}/_build/html ${generate_doc_folder}/
```

Reply:
After discussing with @javmorcas : 
When clean is called remove:
- /doc
- /srcDoc/_build